### PR TITLE
Add information about in-toto to the SLSA website

### PR DESCRIPTION
**Pull request by adityasaky**
This PR adds some information about in-toto and its relationship to SLSA.

### CODE REVIEW

**Comment by adityasaky** [comment]
Just noticed a rendering issue because of the link in the FAQ title. Fix incoming.
image

**Comment by adityasaky** [comment]
Fixed!

File: docs/spec/v0.1/faq.md [hunk 1]
```diff
@@ -67,3 +67,24 @@ verified reproducible builds are one option for implementing the requirements.
...
 [Hermetic, Reproducible, or Verifiable?](https://sre.google/static/pdf/building_secure_and_reliable_systems.pdf#page=357)
+
+## Q: How does SLSA relate to in-toto?
+
+[in-toto](https://in-toto.io/) is a framework to secure software supply chains
+hosted at the [Cloud Native Computing Foundation](https://cncf.io/). The in-toto
+specification provides a generalized workflow to secure different steps in a
+software supply chain.
+[in-toto attestations](https://github.com/in-toto/attestation) are how SLSA
+Provenance and other attributes are expressed. Thus, in-toto can be thought of
```
Comment (MarkLodato) [inline docs/spec/v0.1/faq.md:78]: in-toto is a recommendation, not a requirement

(same in the v1.0 page)

```suggestion
Provenance and other attributes are recommended to be expressed. Thus, in-toto can be thought of
```

File: docs/index.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ testimonials:
             <div class="text w-full md:w-1/2">
                 <h4 class="h2 mb-8">Levels of assurance</h4>
                 <p>SLSA levels are like a common language to talk about how secure software, supply chains and their component parts really are. From source to system, the levels blend together industry-recognized best practices to create four compliance levels of increasing assurance.
-                These look at the builds, sources and dependencies in open source or commercial software. Starting with easy, basic steps at the lower levels to build up and protect against advanced threats later, bringing SLSA into your work means prioritized, practical measures to prevent unauthorized modifications to software, and a plan to harden that security over time.</p>
+                These look at the builds, sources and dependencies in open source or commercial software. Starting with easy, basic steps at the lower levels to build up and protect against advanced threats later, bringing SLSA into your work means prioritized, practical measures built on in-toto attestations to prevent unauthorized modifications to software, and a plan to harden that security over time.</p>
```
Comment (MarkLodato) [inline docs/index.md:100]: nit: I'm not sure this makes sense here. This front page, and this paragraph in paragraph in particular, is very abstract, so it seems odd to mention a particular technology ehre. Maybe revert this one particular change?

**Review by MarkLodato — CHANGES_REQUESTED** [review]
Thanks!

File: docs/index.md (hunk 2 above)
Comment (adityasaky) [reply docs/index.md]: Reverted.

File: docs/spec/v0.1/faq.md (hunk 1 above)
Comment (adityasaky) [reply docs/spec/v0.1/faq.md]: Took a second pass because the original sentence sounded odd when I looked at it again. WDYT?

**Review by MarkLodato — APPROVED** [review]

**Pull request merged**